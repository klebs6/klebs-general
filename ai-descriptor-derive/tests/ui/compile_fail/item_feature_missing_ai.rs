// ---------------- [ File: ai-descriptor-derive/tests/ui/compile_fail/item_feature_missing_ai.rs ]
#![allow(unused_imports)]
use ai_descriptor_derive::*;
use ai_descriptor_trait::*;

#[derive(ItemFeature)]
enum PotionEffect {
    Invisibility, // Missing #[ai("...")]
    #[ai("Gives you super strength.")]
    Strength,
}

fn main() {}
