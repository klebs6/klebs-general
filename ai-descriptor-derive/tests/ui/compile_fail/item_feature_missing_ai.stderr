error: Unit variants must have an #[ai("...")] attribute
 --> tests/ui/compile_fail/item_feature_missing_ai.rs:8:5
  |
8 |     Invisibility, // Missing #[ai("...")]
  |     ^^^^^^^^^^^^
